radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/forks/modules/MainnetForksModule.java

```diff
@@ -190,9 +190,9 @@ ForkBuilder olympiaRadixProtocolFork1() {
             (long) 1024 * 1024, // 1MB max user transaction size
             OptionalInt.of(50), // 50 Txns per round
             10_000, // Rounds per epoch
-            500, // Two weeks worth of epochs
+            500, // Validator fee increases take effect after 500 epochs (roughly two weeks)
```

Comment:
Maybe it is worth adding sth like "given that currently an epoch lasts  x mins".

Reply:
Sure - again, would you be happy if we remember this for the next fork we create? I'm not sure merging a release branch into main is the right time to be making these changes?